docroot/sites/all/themes/ilr_theme/scss/layout/_certificate_programs.scss

```diff
@@ -0,0 +1,72 @@
+body.node-type-certificate-programs {
+  .submitted {
+    display: none;
+  }
+
+  #page-title {
+    display: none;
```

Comment:
What's happening here? Is this because the title was moved? If so, unsetting the page title should be handled in code, not CSS.